src/lib/ios_blur_view_emulation.js

```diff
@@ -0,0 +1,37 @@
+import { Component, PropTypes } from 'react';
+import { View, Platform, requireNativeComponent } from 'react-native';
+
+// This are helper functions that are used to simulate the blur effect on iOS
+// on other platforms (Android and web)
+
+const OVERLAY_COLORS = {
+  light: 'rgba(255, 255, 255, 0.2)',
+  xlight: 'rgba(255, 255, 255, 0.75)',
+  dark: 'rgba(16, 12, 12, 0.64)',
```

Comment:
@ndbroadbent FYI, I determined the following colors from the view debugger in Xcode:
```js
dark: 'rgba(28, 28, 28, 0.73)'
light: 'rgba(255, 255, 255, 0.3)'
xlight: 'rgba(247, 247, 247, 0.8)'
```